packages/backend/src/errors/generate-error-email.ts

```diff
@@ -0,0 +1,53 @@
+import { DateTime } from 'luxon'
+
+import { sendEmail } from '@/helpers/send-email'
+
+const MAX_LENGTH = 80
+
+function truncateFlowName(flowName: string) {
+  return flowName.length > MAX_LENGTH
+    ? flowName.slice(0, MAX_LENGTH) + '...'
+    : flowName
+}
+
+export function createBodyErrorMessage(flowName: string): string {
+  const currDateTime = DateTime.now().toFormat('MMM dd yyyy, HH:mm:ss')
```

Comment:
nit: can we format as AM/PM instead of 24H, to me 24H is very confusing

Reply:
hmm i thought 24h is clearer but okkk